packages/Chatdown/bin/chatdown.js

```diff
@@ -110,9 +112,15 @@ async function processFiles(inputDir, outputDir) {
  */
 async function runProgram() {
     const args = minimist(process.argv.slice(2));
+    
+    let latest = await latestVersion(package.name);
+    if (semver.gt(latest, package.version)) {
+      process.stderr.write(chalk.default.yellowBright(`\nNew version ${latest} is available to install.\n\n`));
```

Comment:
Consider replacing this with these lines (reviewed text with design). 

        process.stderr.write(chalk.default.white(`\n     Update available`));
        process.stderr.write(chalk.default.gray(` ${pjson.version}`));
        process.stderr.write(chalk.default.white(` -> `));
        process.stderr.write(chalk.default.greenBright(`${latest}\n`));
        process.stderr.write(chalk.default.white(`       Run `));
        process.stderr.write(chalk.default.blueBright(`npm i -g ${pjson.name}`));
        process.stderr.write(chalk.default.white(` to update.\n\n`));

![screen shot 2018-10-10 at 2 46 38 pm](https://user-images.githubusercontent.com/20918136/46767942-746b8200-cc9b-11e8-8b47-36d00fb3aa4a.png)